tests/test_pipreqs.py

```diff
@@ -559,6 +603,28 @@ def test_parse_requirements_handles_file_not_found(self, exit_mock):
 
             self.assertEqual(printed_text, "File xpto was not found. Please, fix it and run again.")
 
+    def test_ignore_notebooks(self):
+        """
+        Test the --ignore-notebooks parameter
+        """
+        pipreqs.init(
+            {
+                "<path>": self.project_with_notebooks,
+                "--savepath": None,
+                "--use-local": None,
+                "--force": True,
+                "--proxy": None,
+                "--pypi-server": None,
+                "--print": False,
+                "--diff": None,
+                "--clean": None,
+                "--mode": None,
+                "--ignore-notebooks": True,
+            }
+        )
+        assert os.path.exists(self.requirements_notebook_path) == 1
+        assert os.path.getsize(self.requirements_notebook_path) <= 1
```

Comment:
what does this check exactly?

Reply:
It just checks if the `requirements.txt` file generated by this test is created and is empty.